2022-11-17T00:08:24.1185083-03:00 0HMM893QO71PC:00000013 [ERR] Erro ao Realizar Login!Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index') (6fde2012)
2022-11-17T00:09:19.6775908-03:00 0HMM893QO71PE:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:10:47.6310771-03:00 0HMM893QO71PG:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:11:12.6874756-03:00 0HMM893QO71PH:00000011 [ERR] Erro ao Realizar Login!Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index') (6fde2012)
2022-11-17T00:11:45.2345697-03:00 0HMM895PJO5FB:00000001 [ERR] Erro ao Realizar Login!Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index') (6fde2012)
2022-11-17T00:13:05.3968254-03:00 0HMM895PJO5FE:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:14:36.7922508-03:00 0HMM896Q1DD66:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:26:41.6577520-03:00 0HMM896Q1DD6A:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:33:40.9513017-03:00 0HMM89HAJOV4R:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:45:02.9665932-03:00 0HMM89NMSAVHO:00000011 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T00:58:17.7584578-03:00 0HMM89S1JKB63:00000023 [ERR] Erro ao Realizar Login!Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index') (6fde2012)
2022-11-17T00:59:05.7019214-03:00 0HMM89S1JKB63:00000055 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T01:32:18.1137653-03:00 0HMM8AHNOLUJ5:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
MySql.Data.MySqlClient.MySqlException (0x80004005): The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior.
   at MySqlConnector.Core.ServerSession.TryResetConnectionAsync(ConnectionSettings cs, IOBehavior ioBehavior, CancellationToken cancellationToken) in C:\projects\mysqlconnector\src\MySqlConnector\Core\ServerSession.cs:line 475
   at MySqlConnector.Core.ConnectionPool.GetSessionAsync(MySqlConnection connection, IOBehavior ioBehavior, CancellationToken cancellationToken) in C:\projects\mysqlconnector\src\MySqlConnector\Core\ConnectionPool.cs:line 72
   at MySqlConnector.Core.ConnectionPool.GetSessionAsync(MySqlConnection connection, IOBehavior ioBehavior, CancellationToken cancellationToken) in C:\projects\mysqlconnector\src\MySqlConnector\Core\ConnectionPool.cs:line 141
   at MySql.Data.MySqlClient.MySqlConnection.CreateSessionAsync(ConnectionPool pool, Nullable`1 ioBehavior, CancellationToken cancellationToken) in C:\projects\mysqlconnector\src\MySqlConnector\MySql.Data.MySqlClient\MySqlConnection.cs:line 645
   at MySql.Data.MySqlClient.MySqlConnection.OpenAsync(Nullable`1 ioBehavior, CancellationToken cancellationToken) in C:\projects\mysqlconnector\src\MySqlConnector\MySql.Data.MySqlClient\MySqlConnection.cs:line 312
   at MySql.Data.MySqlClient.MySqlConnection.Open() in C:\projects\mysqlconnector\src\MySqlConnector\MySql.Data.MySqlClient\MySqlConnection.cs:line 280
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnection(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.Open(Boolean errorsExpected)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlRelationalConnection.Open(Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.BeginTransaction(IsolationLevel isolationLevel)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.BeginTransaction()
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(DbContext _, ValueTuple`2 parameters)
   at Pomelo.EntityFrameworkCore.MySql.Storage.Internal.MySqlExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at Killar.Models.UsersService.AddUser(Users newUser) in D:\Workspace\Killar2.0\Models\UsersService.cs:line 18
   at Killar.Controllers.UsersController.RegisterUser(Users newRegisterUser) in D:\Workspace\Killar2.0\Controllers\UsersController.cs:line 29
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2022-11-17T01:43:00.4783997-03:00 0HMM8AO8TICAT:00000001 [ERR] Erro ao se Cadastrar!The client was disconnected by the server because of inactivity. See wait_timeout and interactive_timeout for configuring this behavior. (8b5cc8a1)
2022-11-17T10:38:52.8982054-03:00 0HMM8K41VH2CR:00000033 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Security.Cryptography.CryptographicException: Padding is invalid and cannot be removed.
   at Internal.Cryptography.UniversalCryptoDecryptor.DepadBlock(Byte[] block, Int32 offset, Int32 count)
   at Internal.Cryptography.UniversalCryptoDecryptor.UncheckedTransformFinalBlock(Byte[] inputBuffer, Int32 inputOffset, Int32 inputCount)
   at Internal.Cryptography.UniversalCryptoTransform.TransformFinalBlock(Byte[] inputBuffer, Int32 inputOffset, Int32 inputCount)
   at Killar.Models.Cryptography.DecryptedText(String encrypted) in D:\Workspace\Killar2.0\Models\Cryptography.cs:line 43
   at AspNetCore.Views_Users_UserProfile.<ExecuteAsync>b__8_0() in D:\Workspace\Killar2.0\Views\Users\UserProfile.cshtml:line 54
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.SetOutputContentAsync()
   at AspNetCore.Views_Users_UserProfile.ExecuteAsync() in D:\Workspace\Killar2.0\Views\Users\UserProfile.cshtml:line 4
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewResultExecutor.ExecuteAsync(ActionContext context, ViewResult result)
   at Microsoft.AspNetCore.Mvc.ViewResult.ExecuteResultAsync(ActionContext context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2022-11-17T10:42:47.2142643-03:00 0HMM8K65L02SJ:00000035 [ERR] Erro ao se Cadastrar!String reference not set to an instance of a String. (Parameter 's') (e2b17663)
2022-11-17T10:42:58.8520796-03:00 0HMM8K65L02SJ:00000057 [ERR] Erro ao Realizar Login!String reference not set to an instance of a String. (Parameter 's') (fe66c00d)
2022-11-17T10:43:06.3965487-03:00 0HMM8K65L02SJ:00000067 [ERR] Erro ao Realizar Login!String reference not set to an instance of a String. (Parameter 's') (fe66c00d)
2022-11-17T10:46:21.8144773-03:00 0HMM8K84O20IB:00000021 [ERR] Erro ao se Cadastrar!String reference not set to an instance of a String. (Parameter 's') (e2b17663)
2022-11-17T10:48:08.4476926-03:00 0HMM8K94DGI5H:00000021 [ERR] Erro ao se Cadastrar!String reference not set to an instance of a String. (Parameter 's') (e2b17663)
2022-11-17T10:58:04.0819987-03:00 0HMM8KEPLI11C:00000011 [ERR] Erro ao Realizar Login!Couldn't connect to server (10802214)
